static/app/components/replays/replayDiff.tsx

```diff
@@ -58,70 +62,61 @@ export default function ReplayDiff({leftTimestamp, replay, rightTimestamp}: Prop
       ) : null}
 
       <Flex gap={space(1)} column>
-        <TabList
-          selectedKey={activeTab}
-          onSelectionChange={tab => setActiveTab(tab as Tab)}
-        >
-          <TabList.Item key={Tab.HTML}>{t('Html Diff')}</TabList.Item>
-          <TabList.Item key={Tab.VISUAL}>{t('Visual Diff')}</TabList.Item>
+        <TabList>
+          <TabList.Item key={DiffType.VISUAL}>{t('Visual Diff')}</TabList.Item>
+          <TabList.Item key={DiffType.HTML}>{t('Html Diff')}</TabList.Item>
```

Comment:
I flipped the order of these, and the default.

The HTML item has some kind of problem refreshing itself on initial render, so it renders blank. This is a little change to avoid having to debug and fix that right now